content/01.abstract.md

```diff
@@ -1,18 +1,20 @@
 ## Abstract {.page_break_before}
 
 Scientific news coverage shapes the public’s view of the current state of scientific findings and legitimizes experts.
-Through researching a story, journalists identify and interview a limited number of sources.
-These sources may come from a journalist’s research or through recommendations by other scientists.
-In either case, unconscious biases may influence who is identified as an expert to interview, possibly skewing the selection of interviewees.
-We analyzed more than 16,000 news articles published by _Nature_ to quantify possible disparities.
+News stories frequently cite and quote a limited number of sources.
+These sources may be identified by the journalist’s research or by recommendations by other scientists.
+In both cases, biases may influence who is identified and ultimately published as an expert.
```

Comment:
Is published the right word here? Maybe `included` or similar?

Reply:
yes, i think that's better. added in https://github.com/greenelab/nature_news_manuscript/pull/4/commits/9a361e80d75fa95da3a767292348dc53fab2fa0d